src/test/java/lotto/domain/LottoTicketsTest.java

```diff
@@ -0,0 +1,86 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class LottoTicketsTest {
+
+    @DisplayName("로또 당첨 번호를 전달하면 일치하는 개수가 몇 개 있는지 알 수 있다.")
+    @Test
+    void winningStatics() {
+        //given
+        List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6);
+        int bonus = 45;
+        LottoWinningNumbers lottoWinningNumbers = LottoWinningNumbers.from("1, 2, 3, 4, 5, 6", "45");
+
+        LottoTicket lottoTicket1 = LottoTicket.from(1, 2, 3, 4, 5, 6); //1등
+        LottoTicket lottoTicket2 = LottoTicket.from(1, 2, 3, 4, 5, 22); //3등
+        List<LottoTicket> lottoTicketList = createLottoListFrom(lottoTicket1, lottoTicket2);
+        LottoTickets lottoTickets = new LottoTickets(lottoTicketList, 2000);
+
+        //when
+        Map<LottoRank, Integer> map = lottoTickets.winningStatics(lottoWinningNumbers);
+
+        //then
+        assertThat(map.get(LottoRank.FIRST)).isEqualTo(1);
```

Comment:
다음과 같이 `map`에서 값을 꺼내지 않고도 검증할 수 있다는 점 단순 참고차 말씀드립니다.
```suggestion
        assertThat(map).containsEntry(LottoRank.FIRST, 1);
```